contrib/ipp/include/ippcore.h

```diff
@@ -80,6 +80,8 @@ IPPAPI( void*, mfxMalloc,  (int length) )
 IPPAPI( void, mfxFree, (void* ptr) )
 
 
+IppStatus mfxInit(void);
```

Comment:
Should that be declared IPPAPI()?

Reply:
For current build mode, this is no-op. But yes, for uniformity let's add IPPFUN/IPPAPI.